.github/workflows/playwright-tests.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         run: pnpm db:generate
 
       - name: Install playwright browsers
-        run: pnpx playwright install --with-deps
+        run: pnpm exec playwright install --with-deps
```

Comment:
```suggestion
        run: pnpm exec playwright install --with-deps chromium
```

That should make it much faster (60s -> 15s -ish)

Reply:
Thank you!